pages/tutorials/elastic_ci_stack_aws.md.erb

```diff
@@ -59,11 +47,19 @@ We’ve created a sample [bash-parallel-example sample pipeline](https://github.
 
 <a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/bash-parallel-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add Bash Example to Buildkite" class="no-decoration" width="160" height="30"></a>
 
-Click “Create Pipeline”. Your pipeline is now ready for its first build. Click "New Build" in the top right and choose a build message (perhaps a little party `\:partyparrot\:`?):
+Click “Create Pipeline”.
+
+If your organization is configured to use the web-based steps editor (default), your pipeline is now ready for its first build. 
+
+If your organization has been upgraded to the [YAML steps editor](https://buildkite.com/docs/tutorials/pipeline-upgrade), you should see a "Choose a Starting Point" wizard. Select "Pipeline Upload" from the list:
+
+<%= image "buildkite-pipeline-upload.png", size: "#{782/2}x#{400/2}", alt: 'Upload Pipeline from Version Control' %>
+
+Click "New Build" in the top right and choose a build message (perhaps a little party `\:partyparrot\:`?):
```

Comment:
Something like this might work, but probably breaks the numbering.

```suggestion
1. Click “Create Pipeline”.

    - If your organization is configured to use the web-based steps editor (default), your pipeline is now ready for its first build. 
    - If your organization has been upgraded to the [YAML steps editor](https://buildkite.com/docs/tutorials/pipeline-upgrade), you should see a "Choose a Starting Point" wizard. Select "Pipeline Upload" from the list:

<%= image "buildkite-pipeline-upload.png", size: "#{782/2}x#{400/2}", alt: 'Upload Pipeline from Version Control' %>

2. Click "New Build" in the top right and choose a build message (perhaps a little party `\:partyparrot\:`?):
```